modules/ROOT/pages/migration-module-spring.adoc

```diff
@@ -38,10 +38,14 @@ To fix this problem in Mule 4, we decided to package our Spring support as a sep
 </mule>
 ----
 
-[TIP]	
+[TIP]
 See xref:connectors::spring/spring-module.adoc[Spring Module] to find an example XML file and also further configuration instructions.
 
-The Spring module doesn't ship with any particular Spring dependency. You can provide whatever ones you need by adding the following to your application's `pom.xml`:
+== Configure Spring Module Dependencies in Mule 4
+
+Because the class-loading mechanism implemented in Mule 4 prevents connectors from accessing classes from other connectors, you need to declare connector dependencies as shared libraries.
```

Comment:
This is not what happens here. 
You should explain that "The spring module does not come with any spring dependency, so, in order for it to work, the user should provide one." Then, "since mule connectors dependencies are isolated with each other and from the application dependencies, any spring dependency that the user needs to make available for the spring connector should be declared as sharedLibraries from the app."